server/controllers/profile.js

```diff
@@ -0,0 +1,136 @@
+const Profile = require("../models/Profile");
+const { cloudinary } = require("../utils/cloudinary");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /profile/create
+// @desc Create new profile
+// @access Public
+exports.createProfile = asyncHandler(async (req, res, next) => {
+  try {
+    const {
+      firstName,
+      lastName,
+      gender,
+      birthday,
+      email,
+      phoneNumber,
+      location,
+      profilePic,
+      description,
+      availability,
+    } = req.body;
+    const profile = await Profile.create({
+      firstName,
+      lastName,
+      gender,
+      birthday,
+      email,
+      phoneNumber,
+      location,
+      profilePic,
+      description,
+      availability,
+    });
+
+    res.status(201).json({
+      profile,
+    });
+  } catch (err) {
+    res.status(500);
+    throw new Error("Something went wrong, please try again");
+  }
```

Comment:
So, the `try/catch` here is actually not needed. Now, in other applications maybe but in this one we utilize `express-async-handler` which actually will catch thrown errors and respond with JSON instead of crashing the application. This actually doesn't give us any context into an error that occurs.

Reply:
Please remove in all applicable cases - now, let me mention. If we're `try/catch` for specific errors that are expected then of course we can keep that in!

Reply:
@moffatethan My apologies, this was actually a mix up from an old version. You have made the correction in profile controllers and I have made the changes already. 
Well noted!